src/mudlet-lua/lua/utf8_filenames.lua

```diff
@@ -300,61 +303,6 @@ local function modify_lua_functions(all_compressed_mappings)
             return orig_io_output(file)
          end
 
-         local orig_lfs_attributes = lfs.attributes
```

Comment:
Why is all of this bring removed?

Reply:
I don't know - that is just what the upstream author has done. Maybe it doesn't work? 🤷‍♂️

Reply:
No it was actually I who added them as part of https://github.com/Mudlet/Mudlet/pull/2709. Let's document this better so we don't remove it by accident.

Reply:
Okay - I've put it back... 😀 

Maybe we should offer it back to the upstream author...

Reply:
Unfortunately the commit was not signed (not showing "Verified") as I am out and about and I forgot to grab my OpenGPG card to allow me to sign it... 🙄

Reply:
Will upstream accept them - that would be the best outcome.